### PR TITLE
docs(caching): create new page for Running in Production

### PR DESCRIPTION
**Pull request by hassankhan**
Story details: https://app.clubhouse.io/cube-dev/story/503

### CODE REVIEW

File: docs/content/Caching/Running-in-Production.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+category: Caching
+menuOrder: 4
+---
+
+Cube.js makes use of two different kinds of cache:
+
+- Redis, for in-memory storage of query results
+- Cube Store for persisting pre-aggregations
+
+Redis is included by default in development mode. In production, we **strongly**
```
Comment (keydunov) [inline docs/content/Caching/Running-in-Production.md:13]: Redis isn't included in the development mode. In the development mode, the queue and cache are done in memory.

**Comment by keydunov** [comment]
@hassankhan LGTM + Minor comment re Redis.

File: docs/content/Caching/Running-in-Production.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+title: Running in Production
+permalink: /caching/running-in-production
+category: Caching
+menuOrder: 4
+---
+
+Cube.js makes use of two different kinds of cache:
+
+- Redis, for in-memory storage of query results
+- Cube Store for persisting pre-aggregations
```
Comment (rpaik) [inline docs/content/Caching/Running-in-Production.md:11]: Should it be "persistent" pre-aggregations?

File: docs/content/Caching/Running-in-Production.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+decoupled from the query processing engine, you can horizontally scale your Cube
+Store cluster for as much concurrency as you require.
+
+In cluster mode, Cube Store runs two kinds of nodes:
+
+- a single **router** node handles incoming client connections, manages database
+  metadata and serves simple queries.
+- multiple **worker** nodes which execute SQL queries
+
+In terms of configuration, both the router and worker nodes **must** specify the
```
Comment (rpaik) [inline docs/content/Caching/Running-in-Production.md:94]: I feel like there's a lot of information (of different env. variables) in this paragraph. Would it be better to organize them in bullet points or even in a table?

Comment (hassankhan) [reply docs/content/Caching/Running-in-Production.md:94]: How about a table?

| Environment Variable | Specify on Router? | Specify on Worker? |
|---|---|---|
| `CUBESTORE_SERVER_NAME` | Yes | Yes |
| `CUBESTORE_META_PORT` | Yes | No |
| `CUBESTORE_WORKERS` | Yes | No |
| `CUBESTORE_WORKER_PORT` | No | Yes |
| `CUBESTORE_META_ADDR` | No | Yes |

File: docs/content/Caching/Running-in-Production.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+```yaml
+version: '2.2'
+services:
+  cubestore_router:
+    restart: always
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_LOG_LEVEL=trace
+      - CUBESTORE_SERVER_NAME=cubestore_router:9999
+      - CUBESTORE_META_PORT=9999
```
Comment (keydunov) [inline docs/content/Caching/Running-in-Production.md:118]: @hassankhan I feel that `CUBESTORE_LOG_LEVEL`, `CUBESTORE_SERVER_NAME`, `CUBESTORE_META_PORT` are all optional config.

File: docs/content/Caching/Running-in-Production.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+      - CUBESTORE_WORKERS=cubestore_worker_1:9001,cubestore_worker_2:9001
+      - CUBESTORE_REMOTE_DIR=/cube/data
+    expose:
```
Comment (keydunov) [inline docs/content/Caching/Running-in-Production.md:121]: @hassankhan do we really need to expose cube store ports? I feel they should not be exposed.

File: docs/content/Caching/Running-in-Production.md (hunk 3 above)
Comment (igorlukanin) [reply docs/content/Caching/Running-in-Production.md:94]: @hassankhan Looks great! I suggest maybe even changing "No" → "—" to make it visually more distinguishable from "Yes".

**Comment by igorlukanin** [comment]
@hassankhan I guess this new page should be cross-linked from the Production Checklist, right?

**Comment by keydunov** [comment]
@igorlukanin Yes, we'll update the deployment section in ~2-3 weeks and will do all the cross links.

File: docs/content/Caching/Running-in-Production.md (hunk 5 above)
Comment (hassankhan) [reply docs/content/Caching/Running-in-Production.md:121]: @keydunov I added them after @ilya-biryukov's review, I believe he said they were necessary. Happy to be corrected, though 👍

**Comment by hassankhan** [comment]
@igorlukanin We already do have a link to this page in the Production Checklist, can definitely improve on it though 👍

**Pull request merged**